src/overview/actions.js

```diff
@@ -64,5 +65,38 @@ export function refreshSearch({loadingIndicator = false}) {
     }
 }
 
+// Getting more results
+export function getMoreResults({loadingIndicator = false, endDate, scrollPosition}) {
+    return async function (dispatch, getState) {
+        const { query, startDate } = ourState(getState())
+
+        if (loadingIndicator) {
+            // Show to the user that search is busy
+            dispatch(showLoadingIndicator())
```

Comment:
like i said when getting for results the state of waiting for results need to be +1, so currently it is only changed by the showLoadingIndicator action... that is why i used it. The purpose is to get the idea and the whole logic most importantly, so the show Loading indicator can be changed by an action that only changes state of waiting for results